model/user.go

```diff
@@ -0,0 +1,28 @@
+package model
+
+import (
+	"golang.org/x/crypto/bcrypt"
+)
+
+type User struct {
+	ID           string
+	Username     string
+	EmailAddress string
+	UserPassword []byte
+	IsAdmin	     bool
```

Comment:
So it's rad that you added this (forward thinking), but you'll need to add it to:

`.sql/01-schema.sql` (You can add this as a boolean and set it's default value to false)
`database/user.go#GetUser` (you'll need to return it and set it in the Scan)
`database/database_test.go` (You will need to adjust the test to expect this)
`database/mockdatabase.go` (You'll need to adjust the Mock method to return the new column too)

PROBABLY to
`.sql/02-data.sql` (potentially make one of the new users an admin, the other one a regular user)

Reply:
Went ahead and did this in: 1bcc29a